designer/client/src/actions/nk/genericAction.ts

```diff
@@ -20,8 +21,8 @@ export interface GenericValidationRequest {
 }
 
 export const validateGenericActionParameters = debounce(
```

Comment:
It's named `validateGenericActionParameters` however, under the hood,  ad-hoc test parameters validation, is it ok?

Reply:
You are right, we should change it :+1: . I think that we should change all `genericAction` to `adhocTesting` or similar. It is because genericAction is not used anywhere else and was created only for this (adhoc) functionality. It is names this way only because we though that it is going to be used in more places - but it is not :upside_down_face:

Reply:
Renamed along with other genericAction files